### ISSUE TITLE
Third docker container instance always exits right away

### ISSUE THREAD

**Issue by melihcelik** [tobert/cassandra-docker#35]
I was trying to create a cassandra cluster with multiple nodes using cassandra-docker and I managed to start up two nodes with following commands:
`docker run --name=n1 -d tobert/cassandra`
`docker exec -it n1 nodetool status` which returns 172.17.0.2 as the IP address and then I run following command to get second node up:
`docker run --name=n2 -d tobert/cassandra -seeds 172.17.0.2`
When I use `nodetool status` after a while, it shows second node's status as UN, so I have a cluster of 2 nodes. Then I run the following command to add yet another node into the cluster:
`docker run --name=n3 -d tobert/cassandra -seeds 172.17.0.2`
The command returns immediately and running the command;
`docker ps -a` provides following output:

> CONTAINER ID        IMAGE               COMMAND                  CREATED              STATUS                      PORTS                                               NAMES
> 6757f27b5281        tobert/cassandra    "/bin/cassandra-docke"   11 seconds ago       Exited (1) 11 seconds ago                                                       n3
> 6b5aad3e6f40        tobert/cassandra    "/bin/cassandra-docke"   About a minute ago   Up About a minute           7000/tcp, 7199/tcp, 9042/tcp, 9160/tcp, 61621/tcp   n2
> a1b2302e4a92        tobert/cassandra    "/bin/cassandra-docke"   About a minute ago   Up About a minute           7000/tcp, 7199/tcp, 9042/tcp, 9160/tcp, 61621/tcp   n1

Basically third node exits with error code 1. I cannot restart the node, it still fails right away.

Below is the output of `docker info` command on my machine:

> Containers: 3
>  Running: 2
>  Paused: 0
>  Stopped: 1
> Images: 1
> Server Version: 1.10.3
> Storage Driver: aufs
>  Root Dir: /mnt/sda1/var/lib/docker/aufs
>  Backing Filesystem: extfs
>  Dirs: 31
>  Dirperm1 Supported: true
> Execution Driver: native-0.2
> Logging Driver: json-file
> Plugins:
>  Volume: local
>  Network: bridge null host
> Kernel Version: 4.1.19-boot2docker
> Operating System: Boot2Docker 1.10.3 (TCL 6.4.1); master : 625117e - Thu Mar 10 22:09:02 UTC 2016
> OSType: linux
> Architecture: x86_64
> CPUs: 1
> Total Memory: 1.956 GiB
> Name: default
> ID: ETGO:SLLZ:7ZCJ:FX6C:TXVV:67VJ:T76Z:EIFC:DLLI:5HHN:5NXJ:L5A7
> Debug mode (server): true
>  File Descriptors: 20
>  Goroutines: 34
>  System Time: 2016-03-21T22:10:13.734763854Z
>  EventsListeners: 0
>  Init SHA1:
>  Init Path: /usr/local/bin/docker
>  Docker Root Dir: /mnt/sda1/var/lib/docker
> Labels:
>  provider=virtualbox

**Comment by tobert** [comment]
Can you try launching the last instance with a volume attached, e.g. --volume /tmp/n3:/data then look in /tmp/n3/logs/system.log. I suspect it's a memory or similar limit causing it to fail.

**Comment by melihcelik** [comment]
Attaching the volume did not help unfortunately, the folder /tmp/n3 didn't get created automatically and even if I created manually, nothing was written to it.

I was also suspecting a memory issue but my efforts to increase the allocated memory for the virtual machine by modifying the quick start terminal did not work out. The VM got created with 2048 MB of memory still and third VM still did not get created. After increasing the memory of the VM through VirtualBox helped, now I can create 3 images. Maybe it would be a helpful hint for Mac users that 2048 MB of memory for virtual machine may not be enough for 3 cassandra docker containers running at the same time.

Thank you very much for the suggestion. Closing the ticket as it's not a cassandra-docker image problem.
